pkg/algorithm/cell.go

```diff
@@ -153,6 +157,7 @@ func NewPhysicalCell(
 			totalGpuNum:            n,
 			usedGpuNumAtPriorities: map[CellPriority]int32{},
 			state:                  cellFree,
+			healthy:                true,
 		},
```

Comment:
Init to false, until it is confirmed to be health from ApiServer?

Reply:
Ah yes! I forgot this...

Reply:
Oh I see. We set all cells to bad in h.initBadNodes